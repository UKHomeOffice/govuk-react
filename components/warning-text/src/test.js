import React from 'react';
import ReactDOM from 'react-dom';
import WarningText from './';

describe(WarningText, () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<WarningText>Example</WarningText>, div);
  });
});
