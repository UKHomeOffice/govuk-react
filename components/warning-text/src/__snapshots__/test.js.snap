// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WarningText to match snapshot 1`] = `
.emotion-7 {
  margin-bottom: 0;
}

@media only screen and (min-width:641px) {
  .emotion-7 {
    margin-bottom: 0;
  }
}

.emotion-5 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  margin-bottom: 0;
}

@media only screen and (min-width:641px) {
  .emotion-5 {
    margin-bottom: 0;
  }
}

.emotion-0 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  height: 35px;
  margin-right: 15px;
  width: 35px;
}

.emotion-2 {
  font-family: "nta",Arial,sans-serif;
  font-size: 16px;
  font-weight: 700;
  line-height: 1.25;
  text-transform: none;
}

@media only screen and (min-width:641px) {
  .emotion-2 {
    font-size: 19px;
    line-height: 1.3;
  }
}

<WarningTextWithKnobs>
  <StyledHoc>
    <WarningText
      className="emotion-7 emotion-4"
    >
      <StyledWarningText
        className="emotion-7 emotion-4"
      >
        <div
          className="emotion-4 emotion-5 emotion-6"
        >
          <IconImportantWrapper>
            <div
              className="emotion-0 emotion-1"
            >
              <IconImportant
                fill="currentColor"
                title="icon important"
              >
                <SVG
                  fill="currentColor"
                  preserveAspectRatio="xMidYMid meet"
                  title="icon important"
                  viewBox="0 0 35.000000 35.000000"
                >
                  <svg
                    fill="currentColor"
                    height="100%"
                    preserveAspectRatio="xMidYMid meet"
                    style={
                      Object {
                        "display": "block",
                      }
                    }
                    version="1.1"
                    viewBox="0 0 35.000000 35.000000"
                    width="100%"
                  >
                    <title>
                      icon important
                    </title>
                    <g
                      transform="translate(0.000000,35.000000) scale(0.100000,-0.100000)"
                    >
                      <path
                        d="M100 332 c-87 -48 -125 -155 -82 -232 48 -87 155 -125 232 -82 87 48
125 155 82 232 -48 87 -155 125 -232 82z m100 -122 c0 -53 -2 -60 -20 -60 -18
0 -20 7 -20 60 0 53 2 60 20 60 18 0 20 -7 20 -60z m0 -111 c0 -12 -7 -19 -20
-19 -19 0 -28 28 -14 43 11 11 34 -5 34 -24z"
                      />
                    </g>
                  </svg>
                </SVG>
              </IconImportant>
            </div>
          </IconImportantWrapper>
          <WarningTextWrapper>
            <strong
              className="emotion-2 emotion-3"
            >
              A very long warning message. This includes a lot of text to give a good representation of a more average length warning. That way you can see more than one line wrapping.
            </strong>
          </WarningTextWrapper>
        </div>
      </StyledWarningText>
    </WarningText>
  </StyledHoc>
</WarningTextWithKnobs>
`;
